WordPress/Classes/Utility/LocalizableStringsUtility.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LocalizableStringsUtility.swift
+//  WordPress
+//
+//  Created by Evangelos Sismanidis on 29.09.17.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+
+import Foundation
+
+enum Strings: String {
+    case title = "Title"
+
+    func loc() -> String {
```

Comment:
`loc` isn't a meaningful method name. First thing that comes to mind is `location`, not localized